app/pods/publications/publication/publishpreview/controller.js

```diff
@@ -1,16 +1,68 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
 export default class PublicationPublishPreviewController extends Controller {
+  // properties for making the design
+  @tracked withdrawn = true;
+  @tracked testDate = new Date();
+  @tracked panelCollapsed = false;
+  @tracked panelIcon = 'chevron-down'
+
   get publishPreviewActivities() {
-    console.log('MODEL', this.model);
     const publishPreviewActivities = this.model.publishPreviewActivities.map((activity) => activity);
-    console.log('ACTIVITIES', publishPreviewActivities);
     return publishPreviewActivities;
   }
 
+  // @action
+  // async usedPieces(activity) {
+  //   const pieces = await activity.usedPieces();
+  //   console.log('PIECES', pieces);
+  // }
+
+  @action
+  collapsePanel() {
+    this.panelCollapsed = !this.panelCollapsed;
+    this.panelIcon = ((this.panelIcon === 'chevron-up') ? 'chevron-down' : 'chevron-up');
+  }
+
+  @action
+  async cancelExistingTranslationActivity() {
+    alert('this action is implemented in another ticket');
+  }
+
+  @action
+  async checkExistingTranslationActivity() {
+    alert('this action is implemented in another ticket');
```

Comment:
als je acties op voorhand klaarzet geef je ze best al een correcte naam, zijn geen translation activities

Reply:
in de meeste gevallen is drukproef een publish preview